### PR TITLE
Apple old page

### PR DESCRIPTION
**Pull request by BrianSammit**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          Watch the "Your Verse" film and explore the stories
+        </p>
+      </div>
+    </section>
+    <div class="thumbnails">
+      <img src="images/thumb-1.jpg" alt="" />
+      <img src="images/thumb-2.jpg" alt="" />
+      <img src="images/thumb-3.jpg" alt="" />
+      <img src="images/thumb-4.jpg" alt="" />
+    </div>
```
Comment (expjazz) [inline index.html]: - [ ] These images should be links as is in the original.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      <div>
+        <ul class="ul-2">
+          <li>Site Map |</li>
+          <li>Hot News |</li>
+          <li>RSS Feeds |</li>
+          <li>Media Info |</li>
+          <li>Environments |</li>
+          <li>Job Opportunities |</li>
+          <li>Contact Us |</li>
+          <li><img class="img-footer" src="/images/icone.png" alt="" /></li>
```
Comment (expjazz) [inline index.html]: - [ ] They also need to be a link

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
+      integrity="sha384-xxzQGERXS00kBmZW/6qxqJPyxW3UR0BPsL4c8ILaIWXva5kFi7TxkIIaMiKtqV1Q"
+      crossorigin="anonymous"
+    />
+    <title>Apple old web page clone</title>
+  </head>
+  <body>
+    <section class="main">
```
Comment (expjazz) [inline index.html]: - [ ] You are missing an icon here

| Yours | Original |
|:---:|:---:|
|image | image|

- [ ] Also make them link as is in the original

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes required :recycle: 

#### You did great work on this page. It looks splendid :clap: 

However, there are still minor details in order to check the minimum requirements (check them in the inline comments below).

#### General comments:
- Do not commit the node modules folder. You should use the .gitignore to hide it from git.
image
- You have Linter Errors

Like I said above, the design is great and the code is very clean and easy to understand :ok_hand: 

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:
### General requirements 
- [ ] Add `node_modules` directory  to `.gitignore` file
- [ ] Remove or add to gitignore `.vscode` directory
- [ ] Add link demo to link de mo section with markdown syntax `[link-demo](https://your-link)`
image

### Specific requirements
- Hero section
- [ ] Use `repeat:none;` property to don't repeat your hero background image
image
- Footer
- [ ] Remove extra space 
image

- Bottom

- [ ] Each image should overlap each other

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |
### Improvements for next projects

- [ ] Write a pull request message in the PR box, according to this [example](https://cdn.filestackcontent.com/osvFJEkQHWQsgDe0PJoQ)

image

### Summary

- Please add the node_modules directory to .gitignore file 
- Don't forget to work with inline comments below this comment box.

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: README.md [hunk 4]
```diff
@@ -1 +1,50 @@
...
+
+## Built With
+
+- [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML),
+- [CSS](https://www.w3schools.com/css/),
+- [Fontawesome](https://fontawesome.com/)
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/ositaadimma/Apple-old-page/cc02d9056e7458404f51957be3764b8e976734fc/index.html)
```
Comment (kitpao) [inline README.md]: - [ ] **[OPTIONAL]** As a suggestion, so you don't have to be manually updating this link during your code reviews, use the second link (development link) in githack, that one will automatically load your last version in the same branch :)

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+  display: flex;
+  width: 1349px;
+  height: 200px;
+}
+
+.thumbnails img {
+  display: flex;
+  width: 100%;
+  height: 200px;
+}
```
Comment (kitpao) [inline css/styles.css]: - [ ] Your images are stretched, you can add `object-fit: cover;` rule to make them keep their original proportions

File: index.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+        </ul>
+      </nav>
+      <div class="body-text">
+        <p id="body-text-p1">what will your verse be ?</p>
+        Everyone has something to share. See how people are using <br />
+        ipad in their unique ways to add to the world's story. <br />
+        <p id="body-text-p2">
+          Watch the "Your Verse" film and explore the stories
+          <a href=""><i class="fas fa-chevron-circle-right"></i></a>
+        </p>
```
Comment (kitpao) [inline index.html]: - [ ] Please add an anchor tag `<a>`  to your headers as required by previous TSE, it is not enough to use cursor: pointer;

File: css/styles.css [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+  */
+
+.main {
+  display: flex;
+  width: 100%;
+  height: 538px;
+  justify-content: center;
+  background-image: url(../img/main-body-image.jpg);
+  background-repeat: no-repeat;
+  background-size: 100%;
```
Comment (kitpao) [inline css/styles.css]: - [x] You did great implementing the background-repeat: no-repeat, but now you have a blank space between the hero image and the bottom part, to fix it, use the 'cover' size on the background image
image

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @BrianSammit  and @ositaadimma 

You did a nice job and are almost there, please just make the missing changes required by previous TSE and you will be good to go 👍 

- [x] **[MANDATORY]** I see you added the node_modules folder to your .gitignore as requested, but for it to work, you should follow the next steps:
  * In your local, copy the node_modules folder and put it somewhere else outside your repo
  * Delete the node_modules folder from repo
  * commit and push your changes
  * Then, you can return the copied folder to your repo and it will be ignored by git

- [x] .vscode directory should also be added to gitignore

- [ ] Add a pull request message with a short summary of the changes you introduced in this branch, like in the example:
image

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

File: css/styles.css [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+  background-image: url(../img/main-body-image.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  cursor: pointer;
+}
+
+.ul-1 {
+  display: flex;
+  margin-top: 75px;
+  background-image: url(../img/nav-bar-image.png);
```
Comment (andresporras3423) [inline css/styles.css]: Don't use an image, use a gradient for the background of the navbar.

File: css/styles.css [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+  list-style: none;
+}
+
+#body-text-p2 {
+  font-size: 24px;
+  margin: 20px 0;
+  cursor: pointer;
+}
+
+#body-text-p2 a {
```
Comment (andresporras3423) [inline css/styles.css]: Don't overuse ID selectors

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @ositaadimma @BrianSammit** 
Good job with your new submission for this project :muscle:  :muscle:
There are some improvements to do before you can go ahead:
- [ ] your footer doesn't have the USA flag icon
- [ ] Don't overuse ID Selectors
- [ ] use gradient for the navbar.
- [ ] your linter highlights some errors
- [ ]  Your live demo looks like this:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: css/styles.css [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+  background-image: url(../img/main-body-image.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  cursor: pointer;
+}
+
+.ul-1 {
+  display: flex;
+  margin-top: 75px;
+  background: linear-gradient(rgb(116, 116, 116) 50%, rgb(107, 107, 107) 50%);
```
Comment (bertil291utn) [inline css/styles.css]: ## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display: flex;
}
```
**Html file**

```
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: css/styles.css [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+
+.fab {
+  font-size: 24px;
+  color: rgb(214, 214, 214);
+}
+
+.fas {
+  color: white;
+  position: relative;
+  right: 50px;
```
Comment (bertil291utn) [inline css/styles.css]: ## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: white;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-white{
  color: white;
}
```
**Html file**

```
<div class="color-white">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to follow CSS best practices you required to do these changes, listed inline comments:

### Optional
- [ ] Use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### Summary

-  Don't forget to work with inline comments below this comment box.

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: index.html [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+          <li>
+            <input class="search-input" type="search" />
+          </li>
+          <li><i class="fas fa-search color-white"></i></li>
+        </ul>
+      </nav>
+      <div class="body-text color-white">
+        <p class="body-text-p1">what will your verse be ?</p>
+        Everyone has something to share. See how people are using <br />
+        ipad in their unique ways to add to the world's story. <br />
```
Comment (kitpao) [inline index.html]: - [ ] You are still missing an anchor tag `<a>` for this text

File: css/styles.css [hunk 13]
```diff
@@ -0,0 +1,178 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.flex {
+  display: flex;
+}
```
Comment (kitpao) [inline css/styles.css:9]: - [ ] You are repeating `display: flex;` in lines 57, 92 and 98 in this file for .ul-1 a, .thumbnails a, and .thumbnails img. The objective of doing this special class .flex is that you don't repeat the same rule, to solve this, just go to the HTML elements you are referring to and add the class "flex"

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello Team,

Nice work! you only have 4 mandatory changes pending from previous TSEs. Kindly read them below 🔽 

- [ ] You are still missing the pull request message with a short summary of the changes you introduced in this branch, just edit your pull request.

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Comment by BrianSammit** [comment]
This pull request is to add an old Apple page to the master branch, we made this Apple page with background image, a navbar with background: linear-gradient property and using special class .flex and color .color-white.

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Approved ✅

## You have done an incredible job making this project 👏🧨💪! Just perfect 💯! 

### You have met all the requirements of the project. Go ahead and merge it! Onto the next project 🚀 !

Happy coding 👨‍💻 

<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

**Pull request merged**